README.md

```diff
@@ -45,20 +45,20 @@ skill.onState('my-state', (alexaEvent) => {
 })
 ```
 
-### Logging variables
+### Logging custom variables
 You can also add additional values which will be logged along with the state custom event
 ```javascript
 skill.onState('my-state', (alexaEvent) => {
-  alexaEvent.ama.variables.myVariable = 'hello';
+  alexaEvent.ama.logEvents('response', 'hello');
```

Comment:
why the change in the interface?

Reply:
This would be part of a Custom event:
```
Custom
response: hello
```

Reply:
I just realized the documentation mentions a lot the word `events` and it'd be great to `logEvents` and not just `log`. Since the name `log` sounds like it will write a log somewhere, like the debug or console.log methods.